src/display/editor/tools.js

```diff
@@ -145,6 +171,14 @@ class ImageManager {
     return this.getFromUrl(data.url);
   }
 
+  getSVGUrl(id) {
+    const data = this.#cache.get(id);
+    if (!data?.isSvg) {
+      return null;
+    }
+    return data.svgrl;
```

Comment:
This is always undefined, so I think that you meant the following :-)
```suggestion
    return data.svgUrl;
```